csharp/src/Drivers/Apache/Spark/SparkHttpConnection.cs

```diff
@@ -136,6 +146,33 @@ protected override void ValidateOptions()
                     ? connectTimeoutMsValue
                     : throw new ArgumentOutOfRangeException(SparkParameters.ConnectTimeoutMilliseconds, connectTimeoutMs, $"must be a value of 0 (infinite) or between 1 .. {int.MaxValue}. default is 30000 milliseconds.");
             }
+
+            // Parse retry configuration parameters
+            Properties.TryGetValue(SparkParameters.TemporarilyUnavailableRetry, out string? tempUnavailableRetryStr);
```

Comment:
Best practice to check the return value of `TryGetValue` and not assume that the output is set to a particular value when it returns false.